### ISSUE TITLE
Batch mode: Stop after completion

### ISSUE THREAD

**Issue by edontes** [SAP/kafka-connect-sap#91]
Hello,
We have a bunch of SAP tables to import.
We need to use batch mode for this. But when using the batch mode it keeps on importing data continuously.
Is there a way to stop after importing the data except using incrementing mode, please?

**Comment by elakito** [comment]
There is no only-once execution option. We could probably add an option to limit the polling count. But until this option becomes available, another workaround would be to use a very long poll.interval and stop the connector using REST API when the first round appears to be complete.

**Comment by elakito** [comment]
I saw that there was an issue when pausing the connector using the REST API. It polled once more. This was fixed with https://github.com/SAP/kafka-connect-sap/pull/93. I think using this pausing the connector or deleting the connector after the first poll is executed would be the best approach for your particular case for now. Adding an repeatCount option would lead to confusion because the task will still be shown as running while it is internally stopped.
